src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseAdmReplyService.java

```diff
@@ -264,6 +267,26 @@ private void collectClaimants(CaseData caseData, List<TSEAdminEmailRecipientsDat
         }
     }
 
+    private boolean isClaimantResponseRequired(CaseData caseData) {
+        return CASE_MANAGEMENT_ORDER.equals(caseData.getTseAdmReplyIsCmoOrRequest())
+                ? YES.equals(caseData.getTseAdmReplyCmoIsResponseRequired())
+                    && (BOTH_PARTIES.equals(caseData.getTseAdmReplyCmoSelectPartyRespond())
+                        || CLAIMANT_TITLE.equals(caseData.getTseAdmReplyCmoSelectPartyRespond()))
+                : YES.equals(caseData.getTseAdmReplyRequestIsResponseRequired())
+                    && (BOTH_PARTIES.equals(caseData.getTseAdmReplyRequestSelectPartyRespond())
+                        || CLAIMANT_TITLE.equals(caseData.getTseAdmReplyRequestSelectPartyRespond()));
+    }
+
+    private boolean isRespondentResponseRequired(CaseData caseData) {
+        return CASE_MANAGEMENT_ORDER.equals(caseData.getTseAdmReplyIsCmoOrRequest())
+                ? YES.equals(caseData.getTseAdmReplyCmoIsResponseRequired())
+                    && (BOTH_PARTIES.equals(caseData.getTseAdmReplyCmoSelectPartyRespond())
+                        || RESPONDENT_TITLE.equals(caseData.getTseAdmReplyCmoSelectPartyRespond()))
+                : YES.equals(caseData.getTseAdmReplyRequestIsResponseRequired())
+                    && (BOTH_PARTIES.equals(caseData.getTseAdmReplyRequestSelectPartyRespond())
+                        || RESPONDENT_TITLE.equals(caseData.getTseAdmReplyRequestSelectPartyRespond()));
+    }
```

Comment:
Could these methods be combined? Maybe pass CLAIMANT_TITLE in as a parameter and reference this genericized version instead